package.json

```diff
@@ -91,5 +93,9 @@
       "prettier --config .prettierrc.json --ignore-path .prettierignore --write .",
       "eslint --config .eslintrc.json --ignore-path .eslintignore . --fix"
     ]
+  },
+  "dependencies": {
+    "amqplib": "^0.10.3",
+    "chai-as-promised": "^7.1.1"
```

Comment:
Should these be here? Especially, the amqp dependencies should be removed given that that module is removed entirely.

Reply:
Already removed it, noticed it too